src/page/personal-information.page.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable class-methods-use-this */
+import {
+  $, $$, by, browser, ElementFinder, element, ExpectedConditions
+} from 'protractor';
+
+export class PersonalInformationPage {
+  private firstNameInput: ElementFinder;
+
+  private lastNameInput: ElementFinder;
+
+  private submitButton: ElementFinder;
+
+  constructor() {
+    this.firstNameInput = $('[name="firstname"]');
+    this.lastNameInput = $('[name="lastname"]');
+    this.submitButton = $('[name="submit"]');
```

Comment:
Remember that step 2 of section 19, says that for this case, we should avoid the usage of cssSelectors. So, try to use a different selector :)